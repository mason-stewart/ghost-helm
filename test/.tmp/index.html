<!DOCTYPE html>
<html>
  <head>
    <title>Welcome to The Iron Yard</title>
    <meta name="description" content="We run an intensive code school, a startup accelerator, kids's programming classes, and organize tech conferences and events."/>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- Place favicon.ico and apple-touch-icon.png in the root directory-->
    <!-- build:css /styles/vendor.css-->
    <!-- bower:css-->
    <link rel="stylesheet" href="/bower_components/ghost-shield/dist/stylesheets/ghost-shield.css">
    <link rel="stylesheet" href="/bower_components/ghost-shield/dist/webfonts/ss-social.css">
    <!-- endbower-->
    <!-- endbuild-->
    <!-- build:css({test/.tmp,test/fixtures}) /styles/main.css-->
    <link rel="stylesheet" href="/styles/main.css">
    <!-- endbuild-->
    <!-- build:js /scripts/vendor/modernizr.js-->
    <script src="/bower_components/modernizr/modernizr.js"></script>
    <!-- endbuild-->
    <script type="text/javascript" src="//use.typekit.net/zza7guc.js"></script>
    <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
  </head>
  <body>
    <div class="container-fluid gray-dark-bg">
      <div class="header-dimmer gray-dark-bg">
        <div class="container">
          <div class="header">
            <div class="logo"><a href="http://theironyard.com"><img src="/images/iron-yard-logo.svg" style="preserveAspectRatio=MinYMin slice"/></a></div>
            <div class="nav">
              <div class="primary-nav">
                <ul>
                  <li><a href="/locations">Locations</a></li>
                  <li><a href="/academy">Academy</a></li>
                  <li><a href="/accelerator">Accelerator</a></li>
                  <li><a href="/events">Events</a></li>
                </ul>
              </div>
              <div class="secondary-nav">
                <ul>
                  <li><a href="/about">About</a></li>
                  <li><a href="/jobs">Jobs</a></li>
                  <li><a href="http://news.theironyard.com">News</a></li>
                  <li><a href="http://blog.theironyard.com">Blog</a></li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div style="background:url(/images/home/home-hero.jpg)   no-repeat center/cover transparent;box-shadow: none;" class="image-header image-header-home">
        <div class="container">
          <div class="sub-nav">
            <div class="col-xs-12">
              <h2 style="margin-bottom:-20px"></h2>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-md-8 col-md-offset-2 col-xs-12">
          <h1>Welcome to The Iron Yard</h1>
          <p>We are committed to training the highest-quality developers and startups in the world, both in skill and in character. Our mission is to find people who want to pursue the craft and life-long adventure of technology, teach them the tools of the trade, and then release them into the world with the drive and capability to make a difference. We believe that passionate, talented people who love what they do will create great families, jobs, companies and solutions to the world’s problems. </p>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="background-image: url('/images/home/home-academy.jpg'); margin-top: 20px; height: 300px; box-shadow: inset 0 -90px 60px -14px rgba(0,0,0,.65);" class="image-header">
        <div style="height: 300px;" class="row">
          <div style="bottom: -130px;" class="sub-nav">
            <div class="col-md-8 col-md-offset-2 col-xs-12">
              <h2 style="margin-bottom: 20px; margin-left: 30px;">Intensive Code School</h2>
              <div style="margin-left: 30px;" class="triangle-cutout"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-md-6 col-md-offset-2 col-xs-12"><a style="text-decoration:none; color:#39474e">
            <h3>Learn to code. <br>Life's too short for the wrong career. </h3></a>
          <p style="margin-bottom: 0px;">The Iron Yard Academy teaches intensive, three-month courses that mentor you into a professional junior-level programmer. Our program goes far beyond tactical skills and teaches you to think like a software engineer. Programming is a lifelong study, and our process shows you to learn how to learn so you remain highly valuable for the rest of your career.</p>
        </div>
        <div style="padding-top: 45px;" class="col-md-2 col-xs-12"><a href="/academy" class="btn">Learn More</a>
          <p><em>We offer courses in cities across the country. Find a school in a city near you.</em></p>
        </div>
        <div class="col-md-8 col-md-offset-2 col-xs-12">
          <hr>
          <div style="margin-top: 30px;" class="row">
            <div style="margin-bottom: 30px;" class="col-md-6">
              <h4 style="padding-bottom: 0px;"><a href="/academy/front-end-engineering" class="green-text"><img src="/images/locations/icons/web-design-icon.png" style="height:24px"> <strong>Web Design</strong></a></h4>
              <p>Learn Web Design from the ground up, including ideation, UI and UX design, front-end code and a whole lot more. At the end of class you'll have the skills to understand and add value in almost any digital design role.</p><a href="/academy/web-design" class="btn hover-purple">Learn More</a>
            </div>
            <div style="margin-bottom: 30px;" class="col-md-6">
              <h4 style="padding-bottom: 0px;"><a href="/academy/mobile-engineering" class="green-text"><img src="/images/locations/icons/mobile-icon.png" style="height:24px"> <strong>Mobile Engineering</strong></a></h4>
              <p>Learn iOS app development using Xcode 5 and a whole lot of Objective-C. At the end of the class you’ll have the skills to create beautiful, fully functional iOS apps and the foundation to pursue building Mac apps.</p><a href="/academy/mobile-engineering" class="btn hover-blue">Learn More</a>
            </div>
          </div>
          <div style="margin-top: 30px;" class="row">               
            <div style="margin-bottom: 30px;" class="col-md-6">
              <h4 style="padding-bottom: 0px;"><a href="/academy/front-end-engineering" class="green-text"><img src="/images/locations/icons/js-icon.png" style="height:24px"> <strong>Front End Engineering</strong></a></h4>
              <p>Learn to build in-browser apps in the Front End Engineering program using HTML, CSS and JavaScript and basic server-side tech. At the end of the class you’ll have the skills to create beautiful, fully functional websites and web applications.</p><a href="/academy/front-end-engineering" class="btn hover-yellow">Learn More</a>
            </div>
            <div style="margin-bottom: 30px;" class="col-md-6">
              <h4 style="padding-bottom: 0px;"><a href="/academy/rails-engineering" class="green-text"><img src="/images/locations/icons/rails-icon.png" style="height:24px"> <strong>Rails Engineering</strong></a></h4>
              <p>Learn Ruby on Rails, one of the most popular server-side frameworks in the world. At the end of class you’ll know how to build rock-solid servers, manage databases and know the basics of front end app development.</p><a href="/academy/rails-engineering" class="btn hover-red">Learn More</a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="background-image: url('/images/home/home-accelerator.jpg'); margin-top: 20px; height: 300px; box-shadow: inset 0 -90px 60px -14px rgba(0,0,0,.65);" class="image-header">
        <div style="height: 300px;" class="row">
          <div style="bottom: -130px;" class="sub-nav">
            <div class="col-md-8 col-md-offset-2 col-xs-12">
              <h2 style="margin-bottom: 20px; margin-left: 30px;">Startup Accelerator</h2>
              <div style="margin-left: 30px;" class="triangle-cutout"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="margin-bottom: 60px;" class="row">
        <div class="col-md-6 col-md-offset-2 col-xs-12"><a style="text-decoration:none; color:#39474e">
            <h3>We’re interested in building strong, sustainable businesses that add real value to customers' lives.</h3></a>
          <p style="margin-bottom: 0px;">Speaking the same language? We’d love for you to join us. Whether your goal is a Series A round, building a team and bootstrapping, or utilizing Kickstarter, we have a place for you. We run three programs a year, a digital health program and an open tech program and a green tech program.</p>
        </div>
        <div style="padding-top: 45px;" class="col-md-2 col-xs-12"> <a href="/accelerator" class="btn">Learn More</a>
          <p><em>We run mentorship-driven programs focused on Digital Health, Green Tech and Education.</em></p>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="background-image: url('/images/home/home-kids.jpg'); margin-top: 20px; height: 300px; box-shadow: inset 0 -90px 60px -14px rgba(0,0,0,.65);" class="image-header">
        <div style="height: 300px;" class="row">
          <div style="bottom: -130px;" class="sub-nav">
            <div class="col-md-8 col-md-offset-2 col-xs-12">
              <h2 style="margin-bottom: 20px; margin-left: 30px;">Kids Education</h2>
              <div style="margin-left: 30px;" class="triangle-cutout"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="margin-bottom: 60px;" class="row">
        <div class="col-md-6 col-md-offset-2 col-xs-12"><a style="text-decoration:none; color:#39474e">
            <h3>We believe in investing in the local tech economy for the long-run</h3></a>
          <p style="margin-bottom: 0px;">...and that means impacting the next generation. Students in our intensive courses are required to volunteer, teach and run our Kids programs as a part of their involvement in the Academy. (Not only does this develop a giving mindset, but it helps them communicate things they’ve learned—an essential skill as they grow into management positions in their careers.)</p>
        </div>
        <div style="padding-top: 45px;" class="col-md-2 col-xs-12"> <a href="/academy#kids" class="btn">Learn More</a>
          <p><em>Our kids classes range from Scratch and Minecraft to building websites. Find classes in a city near you.</em>  </p>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="background-image: url('/images/home/home-events.jpg'); margin-top: 20px; height: 300px; box-shadow: inset 0 -90px 60px -14px rgba(0,0,0,.65);" class="image-header">
        <div style="height: 300px;" class="row">
          <div style="bottom: -130px;" class="sub-nav">
            <div class="col-md-8 col-md-offset-2 col-xs-12">
              <h2 style="margin-bottom: 20px; margin-left: 30px;">Events</h2>
              <div style="margin-left: 30px;" class="triangle-cutout"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid">
      <div style="margin-bottom: 60px;" class="row">
        <div class="col-md-6 col-md-offset-2 col-xs-12"><a style="text-decoration:none; color:#39474e" href="/events">
            <h3>We've partnered with Unmatched Style to offer some of the finest design and development events in the country. </h3></a>
          <p style="margin-bottom: 0px;">The world doesn't need more conferences, it needs <em>better</em> conferences. We produce high quality events that are worth every dollar you spend on them. Join us for nationally-renowned gatherings like <a href="/events/grok">Grok</a>, <a href="http://unmatchedstyle.com/events">Converge</a> and the brand new <a href="http://outlinesconf.education">Outlines Education conference</a>. </p>
        </div>
        <div style="padding-top: 45px;" class="col-md-2 col-xs-12"> <a href="/events" class="btn">Learn More</a>
          <p><em>From South Carolina to Las Vegas, we bring top brass and young talent together to learn, converse and have a whole lotta fun.</em></p>
        </div>
      </div>
      <div class="row">
        <div class="horizontal-section gray-light-bg last">
          <div class="container">
            <div class="row">
              <div class="col-md-12">
                <div style="margin-bottom:0" class="callout-box">
                  <h2>Who We Are</h2>
                  <hr>
                  <p style="margin-bottom:0">The Iron Yard exists to create exceptional value for people and their ideas through code education, startup accelerators and coworking spaces. Through our broad, deep and growing network of teachers, programmers, designers, entrepreneurs, investors, employers and coworking spaces we have developed a thriving community where serendipity happens en-masse.</p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container-fluid gray-dark-bg">
      <div class="footer">
        <div class="row">
          <div class="logo"><img src="/images/footer-logo.png"></div>
          <div class="footer-nav">
            <ul>
              <li><a href="/locations">Locations</a></li>
              <li><a href="/academy">Academy</a></li>
              <li><a href="/accelerator">Accelerator</a></li>
              <li><a href="/events">Events</a></li>
              <li><a href="/about">About</a></li>
              <li><a href="/jobs">Jobs</a></li>
              <li><a href="http://news.theironyard.com">News</a></li>
              <li><a href="http://blog.theironyard.com">Blog</a></li>
            </ul>
          </div>
          <div class="social-nav">
            <ul>
              <li><a href="http://twitter.com/theironyard" class="ss-icon">twitter</a></li>
              <li><a href="http://facebook.com/theironyard" class="ss-icon">facebook</a></li>
              <li><a href="http://vimeo.com/theironyard" class="ss-icon">vimeo	</a></li>
            </ul>
          </div>
          <div class="copyright">©2014 The Iron Yard. All Rights Reserved   </div>
        </div>
      </div>
    </div>
    <!-- Google Analytics: change UA-XXXXX-X to be your site's ID.-->
    <script type="text/javascript">
      (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
      function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
      e=o.createElement(i);r=o.getElementsByTagName(i)[0];
      e.src='//www.google-analytics.com/analytics.js';
      r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
      ga('create','UA-32355916-2');ga('send','pageview');
      
    </script>
    <!-- build:js /scripts/vendor.js-->
    <!-- bower:js-->
    <script src="/bower_components/jquery/dist/jquery.js"></script>
    <!-- endbower-->
    <!-- endbuild-->
    <!-- build:js /scripts/main.js-->
    <script src="/scripts/main.js"></script>
    <!-- endbuild-->
    <script>var reg = /city=([^&]*)\//g;
var city = reg.exec(window.location.search);
if (city) {
  $('.class-table tr').removeClass('active');
  $('.class-table tr.' + city[1]).first().prependTo('.class-table tbody').addClass('active');
}

    </script>
    <!--browser update warning-->
    <script type="text/javascript">
      var $buoop = {
        vs:{i:9,f:15,o:12.1,s:5.1},
        text: "Your browser is out of date. Please <a href=\"https://browser-update.org/update.html\">Update Your Browser</a> for more security, comfort and the best experience on this site.",
        newwindow: true
      }; 
      $buoop.ol = window.onload; 
      window.onload=function(){ 
        try {if ($buoop.ol) $buoop.ol();}catch (e) {} 
        var e = document.createElement("script"); 
        e.setAttribute("type", "text/javascript"); 
        e.setAttribute("src", "//browser-update.org/update.js"); 
        document.body.appendChild(e); 
      }
      
    </script>
    <!--clicky analytics-->
    <script src="//static.getclicky.com/js"></script>
    <script type="text/javascript">try{ clicky.init(100714900); }catch(e){}</script>
  </body>
</html>