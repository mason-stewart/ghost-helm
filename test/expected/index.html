<!DOCTYPE html>
<html>
  <head>
    <title>Welcome to a Test Suite</title>
    <meta name="description" content="It's cool."/>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="/styles/vendor.css">
    <link rel="stylesheet" href="/styles/main.css">
    <script src="/scripts/vendor/modernizr.js"></script>
  </head>
  <body>
    <div class="container-fluid gray-dark-bg">
      <p>Wow!</p>
    </div>
    <div class="container-fluid">
      <p>Super!</p>
    </div>
    <div class="container-fluid">
      <p>Amazing</p>
    </div>
    <div class="container-fluid">
      <div class="row">
        <h2>Beer!</h2>
      </div>
    </div>
    <div class="container-fluid gray-dark-bg">
      <div class="footer">
        <div class="row">
          <div class="copyright">Â©2014 Programming Is A Cool Job. All Rights Reserved.  </div>
        </div>
      </div>
    </div>
    <script type="text/javascript">
      function(){}
      
    </script>
    <script src="/scripts/vendor.js"></script>
    <script src="/scripts/main.js"></script>
  </body>
</html>